projects/nano-editor.org/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://www.nano-editor.org/dist/v{{ version.major }}/nano-{{ version.raw }}.tar.gz
+
+versions:
+  - 7.2
+
+dependencies:
+  gnu.org/gettext: "*"
+  invisible-island.net/ncurses: ">=6.0"
+
+provides:
+  - bin/nano
+
+build:
+  dependencies:
+    freedesktop.org/pkg-config: "*"
+    gnu.org/automake: "*"
+    gnu.org/make: "*"
+    tea.xyz/gx/cc: "*"
+  working-directory: nano-{{ version.raw }}
+  script: |
+    ./configure $ARGS
+    make install
+  env:
+    ARGS:
+      - --disable-debug
+      - --disable-dependency-tracking
+      - --prefix="{{prefix}}"
+      - --sysconfdir="{{prefix}}"
+      - --enable-color
+      - --enable-extra
+      - --enable-multibuffer
+      - --enable-nanorc
+      - --enable-utf8
+
+test:
+  script: |
+    nano --version
```

Comment:
do a `test $(nano --version) == {{version}}` we've had a lot of packages that don’t contain their version info lately

Reply:
i added a grep test here nano was not returning just the version in `--version`

Reply:
i also noticed something strange. 

at the top the version.raw == 7.2
```
distributable:
  url: https://www.nano-editor.org/dist/v{{ version.major }}/nano-{{ version.raw }}.tar.gz
  strip-components: 1

versions:
  - 7.2
 ```

but when used in test block version.raw became 7.2.0